client/src/main/java/org/dst/client/exception/DstException.java

```diff
@@ -0,0 +1,13 @@
+package org.dst.client.exception;
```

Comment:
Can we move all exceptions to common module?

Reply:
done